### PR TITLE
GSAGH-513 - Broken link is generated by automated doc generator

### PR DESCRIPTION
**Pull request by SandeepArup**

### CODE REVIEW

File: GsaGH/Parameters/5_Results/2_Results/GsaResult.cs [hunk 1]
```diff
@@ -5,18 +5,7 @@
...
-  /// <item><description><see href="https://docs.oasys-software.com/structural/gsa/references/dotnet-api/result-classes.html#element3dresult">3D Element Results</see>: `Displacement` and `Stress`.</description></item>
-  /// <item><description><see href="https://docs.oasys-software.com/structural/gsa/references/dotnet-api/result-classes.html#globalresult">Global Results</see>: `Frequency`, `LoadFactor`, `ModalGeometricStiffness`, `ModalMass`, `ModalStiffness`, `TotalLoad`, `TotalReaction`, `Mode`, `EffectiveInertia`, `EffectiveMass` and `Eigenvalue`.</description></item>
-  /// </list></para>
-  /// <para>All result values from the <see href="https://docs.oasys-software.com/structural/gsa/references/dotnet-api/introduction.html">.NET API</see> have been wrapped in <see href="https://docs.oasys-software.com/structural/gsa/references/gsagh/gsagh-unitnumber-parameter.html">Unit Number</see> and can be converted into different measures on the fly. The Result parameter caches the result values.</para>
-  /// </summary>
+ 
   public class GsaResult : IGsaResult {
```
Comment (RaviKumarGubbala) [inline GsaGH/Parameters/5_Results/2_Results/GsaResult.cs:9]: Why is this deleted!

Comment (RaviKumarGubbala) [reply GsaGH/Parameters/5_Results/2_Results/GsaResult.cs:9]: Ok.. I see.. removed from GsaResult class and moved to IGsaResult

**Review by RaviKumarGubbala — APPROVED** [review]

File: GsaGH/Parameters/5_Results/2_Results/GsaResult.cs (hunk 1 above)
Comment (SandeepArup) [reply GsaGH/Parameters/5_Results/2_Results/GsaResult.cs:9]: Followed same structure in relation to other interface

File: GsaGH/Parameters/5_Results/2_Results/IGsaResult.cs [hunk 2]
```diff
@@ -1,6 +1,19 @@
 ﻿using System.Collections.Generic;
 
 namespace GsaGH.Parameters.Results {
+
```
[the remark this answers is not in the document]
Comment (SandeepArup) [reply GsaGH/Parameters/5_Results/2_Results/IGsaResult.cs]: have manually removed space. Need to configure like auto format in Gsa

**Review by RaviKumarGubbala — APPROVED** [review]

**Review by Chakravarthi-Venigalla — APPROVED** [review]
